src/main/scala/com/microsoft/hyperspace/actions/RefreshDeleteAction.scala

```diff
@@ -91,36 +89,8 @@ class RefreshDeleteAction(
       indexConfig.indexedColumns)
   }
 
-  /**
-   * Compare list of source data files from previous IndexLogEntry to list
-   * of current source data files, validate fileInfo for existing files and
-   * identify deleted source data files.
-   */
-  private lazy val deletedFiles: Seq[String] = {
```

Comment:
Note for reviewers: this method is moved to RefreshActionBase, and `IndexLogEntry.deletedFiles` is added to the result.

Reply:
thanks @apoorvedave1 , noted!